src/i18n/locale-en.json

```diff
@@ -537,6 +537,8 @@
     "menu.sparql.label": "SPARQL",
     "menu.monitor.label": "Monitor",
     "menu.queries.and.updates.label": "Queries and Updates",
+    "menu.fts.label": "Enable full-text search",
```

Comment:
did you forget to move this translations (lines 540 and 541) to the test fixture?